apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -875,14 +873,13 @@ export default function teacherSections(state = initialState, action) {
     }
 
     const lessonExtraSettings = {};
+    if (action.props.scriptId && !action.props.lessonExtras) {
```

Comment:
I'm a bit confused about this case, if `action.props.lessonExtras` is false, we set `lessonExtraSettings.lessonExtras` to true?

Reply:
Hmmm good call out. I was going for if lessonExtras was null so I can update to that

Reply:
Thanks! I'm also slightly confused about why the unit tests were passing before, because it looked like that the case where `action.props.lessonExtras` is false was being tested. That might be worth looking into